### PR TITLE
React & Redux group project - Space Travelers

### PR DESCRIPTION
**Pull request by fbclh**
**Project requirements**

- Config & basic setup
- Redux: Fetch data and update Redux store
- Render UI:lists
- Redux: Write actions and reducers for booking rockets and joining missions
- Redux: Write actions and reducers for canceling rockets and leaving missions
- Render UI: conditional components rendering
- Render UI: My Profile section
- Technical set up:
- - Set up the repository on GitHub and use Gitflow.
- - Set up React and Redux.
- - Set up testing libraries(React Testing Library and Jest).
- - Do not preserve user data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,47 @@
...
-### `npm run build`
+Run
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```sh
+   cd space-travelers
+   npm install
+   npm start
+```
```
Comment (brytebee) [inline README.md:28]: - [ ] **Required:** Kindly add the `Testing` section to your readme to specify the procedure for running the tests. This makes your documentation more professional.

File: src/pages/ProfilePage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// import MissionFilter from '../components/Profile/MissionFilter';
```
Comment (brytebee) [inline src/pages/ProfilePage.js]: - [ ] **Required:** Kindly remove all unused codes from your codebase. It makes code clean, more readable and professional.

File: src/pages/ProfilePage.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import ReservedMissionsList from '../components/Missions/ReservedMissionList';
+import RocketsFilter from '../components/Profile/RocketsFilter';
+
+export default function ProfilePage() {
+  return (
+    <div className="filter-container">
+      {/* <MissionFilter /> */}
```
Comment (brytebee) [inline src/pages/ProfilePage.js]: - [ ] **Required:** Kindly remove as stated above.

File: src/redux/missions/missions.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    mission_name: value.mission_name,
+    description: value.description,
+    reserved: false,
+    wikipedia: value.wikipedia,
+    //
+    // id: key,
+    // missionName: value.mission_name,
+    // missionDescription: value.description,
+    // missionStatus: false,
+    //
```
Comment (brytebee) [inline src/redux/missions/missions.js]: - [ ] **Required:** Kindly remove. The reason previously explained.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @fbclh & @sediqullahbadakhsh👋, 

Good job so far!

## Highlights

- The development follows proper Git flow.👏
- The linter files are set up correctly and pass.👏
- Middleware used appropriately.👏
- All tests 🧪 pass.👏
- Snapshot test created and pass.👏
- The app is deployed.👏
- Reservation works correctly.👏
- React / Redux hooks correctly used.👏

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by fbclh** [comment]
Hi and thanks, based on your feedback I have made the necessary changes.

- Delete all unnecessary comments.
- Edit README to include test instructions.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**